lib/ffmpex.ex

```diff
@@ -195,6 +194,19 @@ defmodule FFmpex do
     unless Enum.member?(contexts, required), do: raise ArgumentError
   end
 
+  defp format_output({:ok, data}) do
+    case Keyword.get(data, :stdout) do
+      nil -> {:ok, nil}
+      binary_list -> {:ok, Enum.join(binary_list)}
+    end
+  end
+
+  defp format_output({:error, data}) do
+    exit_status = Keyword.get(data, :exit_status) |> :exec.status() |> elem(1)
+    stderr = Keyword.get(data, :stderr, []) |> Enum.join(" ") 
```

Comment:
Here we should join with `""` not space. Tested against `System.cmd/3` error output.